tests/end_to_end/test_kakarot.py

```diff
@@ -153,20 +153,80 @@ async def test_should_return_same_as_deployed_address(
     class TestDeployExternallyOwnedAccount:
         async def test_should_deploy_starknet_contract_at_corresponding_address(
             self,
+            starknet: FullNodeClient,
             fund_starknet_address,
             deploy_externally_owned_account,
             compute_starknet_address,
             get_contract,
         ):
-            evm_address = generate_random_evm_address()
+            evm_address = generate_random_evm_address(random.randint(0, 0x5EED))
```

Comment:
otherwise the tx would fail because of a deploy_syscall address collision but the test would pass as the contract was already deployed

Reply:
can't you just do `generate_random_evm_address(0x5EED)`

ie : `generate_random_evm_address` only arg is actually a seed, so why do you need to do a random with this seed to feed the seed?